ui/src/app/components/resources.tsx

```diff
@@ -135,7 +135,8 @@ export class ResourceCardMenu extends React.Component<ResourceCardMenuProps> {
         <ClrIcon shape='ellipsis-vertical' size={21}
                  style={{
                    color: colors.accent,
-                   marginLeft: -9, cursor: 'pointer'
+                   marginLeft: -9,
```

Comment:
nit, but I reserve the right to be more adamant next time 😄: where do these magic numbers for size and margin come from?

Reply:
... in a different PR cause it's actually a lot of changes